src/shared/dropdown-wrapper/dropdown-wrapper.styles.tsx

```diff
@@ -18,27 +18,42 @@ export interface DropdownWrapperStyleProps {
 
 export interface ValueLabelStyleProps {
     truncateType?: TruncateType;
+    $variant?: DropdownVariantType | undefined;
+}
+
+export interface SelectorStyleProps {
+    $variant?: DropdownVariantType | undefined;
 }
 
 // =============================================================================
 // STYLING
 // =============================================================================
 const BORDER_RADIUS = "4px";
+const getHeight = (variant?: DropdownVariantType | undefined) => {
```

Comment:
small nitpick but can you add a newline between 31-32? to separate the two items